netconf_linux.go

```diff
@@ -272,6 +272,10 @@ func setGateway(gateway string) error {
 }
 
 func applyInterfaceConfig(link netlink.Link, netConf InterfaceConfig) error {
+	if err := netlink.LinkSetDown(link); err != nil {
+		return err
+	}
```

Comment:
I'm not too sure this is a good idea.  What if network traffic is currently in progress?